Banking/src/main/java/com/example/Banking/controller/AccountController.java

```diff
@@ -0,0 +1,49 @@
+package com.example.Banking.controller;
+import com.example.Banking.exception.IdNotFoundException;
+import com.example.Banking.model.Account;
+import com.example.Banking.service.AccountService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import java.util.List;
+
+@RestController
+@RequestMapping("/Accounts")
+public class AccountController {
+    @Autowired
+    private AccountService accountService;
+    @GetMapping("/getAllAccounts")
+    public List<Account> getAllAccounts() {
+        return accountService.getEveryAccount();
+    }
+
+    @GetMapping("/getOneAccount/{accountNumber}")
+    public Account getOneAccount(@PathVariable(value = "accountNumber") Long acNumber) throws IdNotFoundException {
+        return accountService.getAccountbyId(acNumber);
+    }
+
+    @PostMapping("/createAccount")
+    public void createAccounts(@RequestBody Account account) {
+        accountService.createAccount(account);
+
+    }
+
+    @DeleteMapping("/deleteAccount/{accountNumber}")
+    public void deleteAccount(@PathVariable(value = "accountNumber") Long acNumber) throws IdNotFoundException {
+
+        accountService.deleteAccountbyId(acNumber);
+
+    }
+
+    @PutMapping("/updateAccount/{accountNumber}")
+    public void updateAccount(@PathVariable(value = "accountNumber") Long acNumber, @RequestBody Account account) throws IdNotFoundException {
```

Comment:
account has accountNumber. I don't think we need to pass accountNumber separately

Reply:
Only If I pass the account number that particular account details will be fetched for changes to be made

Reply:
I have reomoved the account number and performed the update operation. It is working.

Reply:
removed*